### PR TITLE
steam: move documentation to README.voidlinux

### PR DESCRIPTION
**Pull request by cinerea0**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/steam/template [hunk 1]
```diff
@@ -30,4 +30,7 @@ do_install() {
 	# Device support for Steam-related hardware (e.g. controllers over Steam Link)
 	vinstall ${FILESDIR}/60-steam-input.rules 644 usr/lib/udev/rules.d
 	vinstall ${FILESDIR}/60-steam-vr.rules 644 usr/lib/udev/rules.d
+
```
Comment (abenson) [inline srcpkgs/steam/template:33]: This might be pedantic, but get rid of the tab here.

File: srcpkgs/steam/files/README.voidlinux [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+For users of the open source drivers:
+
+	- mesa-dri-32bit
+
+For users of the proprietary NVIDIA driver:
+
+	- nvidia-libs-32bit (latest nvidia package)
+	- nvidia390-libs-32bit (latest previous package)
+	- nvidia340-libs-32bit (latest package for old GPUs)
```
Comment (abenson) [inline srcpkgs/steam/files/README.voidlinux:20]: Drop this line, `nvidia340` is gone(ish)

Comment (abenson) [reply srcpkgs/steam/files/README.voidlinux:20]: (I should have already done this, but it's an opportunity as good as any!)

**Comment by cinerea0** [comment]
As per discussion in https://github.com/void-linux/void-docs/pull/496, instructions currently located in INSTALL.msg have been moved to README.voidlinux, and troubleshooting information previously located in the Void wiki has been moved to that file as well. The INSTALL.msg file now directs users to check the README file.

**Pull request merged**